pkg/app/ops/handler/handler_mock.go

```diff
@@ -1,95 +0,0 @@
-// Code generated by MockGen. DO NOT EDIT.
```

Comment:
The linter warns for this file, but actually this file seems to be not needed.

I noticed the mock for the project store is already created in the `pkg/datastore/datastoretest`.